x-pack/plugins/actions/server/builtin_action_types/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ActionTypeService } from '../action_type_service';
+
+import { actionType as serverLogActionType } from '././server_log';
```

Comment:
@mikecote noticed the `././` oddity here, which I suspect is from a vscode file rename or such.  WIll fix.